### ISSUE TITLE
Support EIP-695 (eth_chainId)

### ISSUE THREAD

**Issue by feuGeneA** [ConsenSys-archive/ganache-cli-archive#682]
* I asked about this on [gitter](https://gitter.im/ConsenSys/truffle): [x]
* Link to my question on gitter: https://gitter.im/ConsenSys/truffle?at=5d94b7c966c8b45122573b35

## Expected Behavior

RPC endpoint should respond to the `eth_chainId` method per EIP-695.

## Current Behavior

I'm using Web3.py's `web3.eth.chainId` interface, and getting the following back from ganache-cli:

```
146 >>> Web3(ganache).eth.chainId
UNEXPECTED EXCEPTION: ValueError({'message': 'Method eth_chainId not supported.', 'code': -32000, 'data': {'stack': 'Error: Method eth_chainId not supported.\n    at u.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:377936)\n    at t (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241567)\n    at a.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:376923)\n    at t (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241567)\n    at s.f.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:370053)\n    at s.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:367879)\n    at t (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241567)\n    at c.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:375937)\n    at t (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241567)\n    at a.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:375384)\n    at t (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241567)\n    at d._handleAsync (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241603)\n    at Timeout._onTimeout (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241028)\n    at ontimeout (timers.js:436:11)\n    at tryOnTimeout (timers.js:300:5)\n    at listOnTimeout (timers.js:263:5)\n    at Timer.processTimers (timers.js:223:10)', 'name': 'Error'}})
Traceback (most recent call last):

  File "/home/gene/.pythonz/pythons/CPython-3.7.2/lib/python3.7/doctest.py", line 1329, in __run
    compileflags, 1), test.globs)

  File "<doctest zero_ex.contract_wrappers[27]>", line 1, in <module>

  File "/home/gene/.pythonz/pythons/CPython-3.7.2/lib/python3.7/site-packages/web3/eth.py", line 106, in chainId
    return self.web3.manager.request_blocking("eth_chainId", [])

  File "/home/gene/.pythonz/pythons/CPython-3.7.2/lib/python3.7/site-packages/web3/manager.py", line 97, in request_blocking
    raise ValueError(response["error"])

ValueError: {'message': 'Method eth_chainId not supported.', 'code': -32000, 'data': {'stack': 'Error: Method eth_chainId not supported.\n    at u.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:377936)\n    at t (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241567)\n    at a.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:376923)\n    at t (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241567)\n    at s.f.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:370053)\n    at s.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:367879)\n    at t (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241567)\n    at c.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:375937)\n    at t (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241567)\n    at a.handleRequest (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:375384)\n    at t (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241567)\n    at d._handleAsync (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241603)\n    at Timeout._onTimeout (/usr/lib/node_modules/ganache-cli/build/ganache-core.node.cli.js:52:241028)\n    at ontimeout (timers.js:436:11)\n    at tryOnTimeout (timers.js:300:5)\n    at listOnTimeout (timers.js:263:5)\n    at Timer.processTimers (timers.js:223:10)', 'name': 'Error'}}
```

## Context
I have some client-agnostic tests, which run against geth and parity, and which depend on the `eth_chainId` RPC method.  Currently I have to hardcode a special case for these tests to also run 
against ganace-cli.

**Comment by davidmurdoch** [comment]
ganache-cli supports `eth_chainId` since version 6.5.0. You may need to update?

**Comment by Melvillian** [comment]
This should be closed, I had the same error and updating to 6.5.0 fixed it.
